pkg/util/kms/kms_vault.go

```diff
@@ -153,13 +152,13 @@ func writeCrtsToFile(secretName string, namespace string, secretValue []byte, en
 	}
 
 	// Generate a temp file
-	file, err := ioutil.TempFile("", "")
+	file, err := os.CreateTemp("", "")
```

Comment:
I think here also we are missing `defer file.Close()`, does that sound right @baum?

Reply:
Great catch, thank you @utkarsh-pro 😀

[Amended](https://github.com/noobaa/noobaa-operator/pull/971/commits/04dca9c6b64338b6de0279cb8ab55289cb03dfaf).